src/libsyntax/tokenstream.rs

```diff
@@ -206,5 +210,1105 @@ impl TokenTree {
                                                          true);
         macro_parser::parse(cx.parse_sess(), cx.cfg(), arg_rdr, mtch)
     }
+
+    /// Check if this TokenTree is equal to the other, regardless of span information
+    pub fn unspanned_eq(&self, other: &TokenTree) -> bool {
```

Comment:
I'd prefer `eq_unspanned`, since it makes it easier to discover when looking for `eq`